Pod/Public/TCCAnimationTileOverlayRenderer.m

```diff
@@ -84,8 +84,9 @@ - (BOOL)canDrawMapRect:(MKMapRect)mapRect zoomScale:(MKZoomScale)zoomScale
             while (tileCol < widthCount) {
                                 
                 MKMapRect localMapRect = MKMapRectMake(mapRect.origin.x + (tileCol * tileSizeForZoomLevel), mapRect.origin.y + (tileRow * tileSizeForZoomLevel), tileSizeForZoomLevel, tileSizeForZoomLevel);
-                
-                TCCAnimationTile *tile = [animationOverlay staticTileForMapRect:localMapRect zoomLevel:cappedZoomLevel];
+
+                //The tile can be nil in a case of low memory
+                __weak __typeof__(TCCAnimationTile *) tile = [animationOverlay staticTileForMapRect:localMapRect zoomLevel:cappedZoomLevel];
```

Comment:
Shouldn't we add a nul check before line ~96 `if (tile.tileImage || tile.failedToFetch) { }` since `tile` can be null.

Reply:
A nil value is the safest way to initialize an object pointer if you don’t have another value to use, because it’s perfectly acceptable in Objective-C to send a message to nil. If you do send a message to nil, obviously nothing happens.

https://stackoverflow.com/questions/2696891/calling-a-method-on-an-uninitialized-object-null-pointer and https://developer.apple.com/library/archive/documentation/Cocoa/Conceptual/ProgrammingWithObjectiveC/WorkingwithObjects/WorkingwithObjects.html#//apple_ref/doc/uid/TP40011210-CH4-SW22

Reply:
My understanding is that the code already checks for nil just below that block.